Database/Stored Procedures/Check/check_podium_performance_festival.sql

```diff
@@ -0,0 +1,33 @@
+
+/*==============================================================*/
+/* DBMS name:		FestiBase									*/
+/* PDM version:		6											*/
+/* Last edited:		8-1-2018									*/
+/* Edited by:		Robert Verkerk								*/
+/* Procedure:		CHECK if the genre of an artist				*/
+/*					is the same as the genre of the podium		*/
+/*==============================================================*/
```

Comment:
Als ik zo naar de banner kijk, gok ik dat het van podium_genre = artist_genre is. Mooet aangepast worden